package.json

```diff
@@ -8,6 +8,8 @@
 		"dev": "next dev",
 		"build": "next build",
 		"start": "next start",
+		"format": "prettier --check --ignore-path .gitignore .",
+		"format:fix": "prettier --write --ignore-path .gitignore .",
```

Comment:
Or maybe `prettier` is formatting the yaml? Can you add `--ignore-path .tekton` as well?

Reply:
Yes that was an intentional format, sorry about that! See my other comment. I'll add `--ignore-path .tekton` to this.